travis/travisRun.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x -e
+
+if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
+    eval "$(/usr/local/bin/pyenv init -)"
+    pyenv global "${PYENV_VERSION}"
+fi
+
+iconv --version
+python --version
+pip --version
+python -u setup.py clean
+CFLAGS="-O0 -g" python -u setup.py build_ext --inplace
+CFLAGS="-O0 -g" PYTHONUNBUFFERED=x make test
+python setup.py bdist_wheel
```

Comment:
I can update this to only build the wheels if the OS is osx if that makes more sense.